src/geo/projection/far_z.js

```diff
@@ -2,14 +2,12 @@
 import {vec3} from 'gl-matrix';
```

Comment:
Just realized the function `furthestPixelDistanceOnSphere` is missing the far plane interpolation during the transition, in native it was set simply as:

```c++
const double t = tr.getProjectionTransitionT();
if (t > 0.0) {
    MercatorProjection mercator;
    const double mercatorPixelDistance = mercator.getFarthestPixelDistance(ctx);
    pixelDistance = util::interpolate(pixelDistance, mercatorPixelDistance, t);
}

// Add a bit extra to avoid precision problems when a fragment's distance is exactly `furthestDistance`
return pixelDistance * 1.01;
```

Reply:
Ah right, I totally missed that! I've added it to `globe.js` because we get the transition's function from there and otherwise it would lead to a circular reference here.

Reply:
That works!